Bootstrap.mak

```diff
@@ -103,6 +103,16 @@ solaris: $(SRC)
 	./build/bootstrap/premake_bootstrap --to=build/bootstrap gmake2
 	$(MAKE) -C build/bootstrap -j`getconf NPROCESSORS_ONLN` config=$(CONFIG)
 
+haiku: $(SRC)
+	$(SILENT) rm -rf ./bin
+	$(SILENT) rm -rf ./build
+	$(SILENT) rm -rf ./obj
+	mkdir -p build/bootstrap
+	$(CC) -o build/bootstrap/premake_bootstrap -DPREMAKE_NO_BUILTIN_SCRIPTS -DLUA_USE_POSIX -DLUA_USE_DLOPEN -D_BSD_SOURCE -I"$(LUA_DIR)" -I"$(LUASHIM_DIR)" $? -lnetwork -lbsd
```

Comment:
what symbols does -lnetwork provide that premake_bootstrap requires?

Reply:
Hmm, none it seems. I just used the same libs as for the rest, but it seems you don't use cURL for bootstrap so it works without. I'll make a fixup.